### PR TITLE
Add a changelog policy

### PR DESCRIPTION
**Pull request by jorisdral**

### CODE REVIEW

File: diff-containers/changelog.d/scriv.ini [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[scriv]
+format = md
+insert_marker = Changelog entries
```
Comment (Jasagredo) [inline diff-containers/changelog.d/scriv.ini:3]: This was used to ignore the header on every changelog and only start when this string is found. I think you should remove it in this case

File: diff-containers/changelog.d/20230208_173935_joris_readmes.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- A bullet item for the Non-Breaking category.
+
+-->
+<!--
+### Breaking
+
+- A bullet item for the Breaking category.
+
+-->
```
Comment (Jasagredo) [inline diff-containers/changelog.d/20230208_173935_joris_readmes.md:20]: I'm not sure what this entry is about? I don't think these first files should be here.

**Review by Jasagredo — CHANGES_REQUESTED** [review]

File: diff-containers/changelog.d/20230208_173935_joris_readmes.md (hunk 2 above)
Comment (jorisdral) [reply diff-containers/changelog.d/20230208_173935_joris_readmes.md:20]: I'll remove them -- I can add "first release" changelog entries when the actual publishing happens

File: diff-containers/changelog.d/scriv.ini (hunk 1 above)
Comment (jorisdral) [reply diff-containers/changelog.d/scriv.ini:3]: I've commented them out for now

**Review by Jasagredo — APPROVED** [review]

**Review by dnadales — APPROVED** [review]

**Pull request merged**